.bazelci/presubmit.yml

```diff
@@ -17,4 +17,130 @@ platforms:
     - "--"
     - "..."
     - "-//tests/docker/..."
-
+  rbe_ubuntu1604:
+    build_targets:
+    - "--"
+    - "//docker/..."
+    - "//skylib/..."
+    - "//container:load"
```

Comment:
try to see if you can do //container/...

Reply:
Done.